app/helpers/object_link_helper.rb

```diff
@@ -115,10 +115,10 @@ def species_fungorum_sf_synonymy(record_id)
   def user_link(user, name = nil)
     if user.is_a?(Integer)
       name ||= "#{:USER.t} ##{user}"
-      link_to(name, User.show_link_args(user))
+      link_to(name, "#{observer_show_user_path}/#{user}")
     elsif user
       name ||= user.unique_text_name
-      link_to(name, user.show_link_args)
+      link_to(name, "#{observer_show_user_path}/#{user.id}")
```

Comment:
This prevents namespacing the link to `show_user`.
See #a0db86e472ec0d561181fcfc7e646a8996bef886

Reply:
I'm guessing the "observer_show_user_path" will get a revision when observer controller is broken up

Reply:
Good point. Yes, that will need revision when ObserverController is updated.